src/test/java/seedu/address/logic/commands/EditCommandTest.java

```diff
@@ -80,20 +80,15 @@ public void execute_noFieldSpecifiedUnfilteredList_success() {
     }
 
     @Test
-    public void execute_filteredList_success() {
+    public void execute_filteredList_failure() {
```

Comment:
Instead of changing this test to a negative test with duplicate modules in a filtered list, could we still keep it as a positive test for successful execution on a filtered list? Cos there's already a number of negative tests below for the same equivalence class 😁  We could edit the tags here instead and assert a successful execution

Reply:
noted!